CMSgov/design-system
# [NO-TICKET] conditional font path for storybook styles

## Summary

In the Storybook webpack config, I created a conditional to set a different font path for design system styles. This is necessary because directory structures between dev & production builds are different (prod is flatter). This also resolves an issue with the new doc site where font files weren't loading.

## How to test

`yarn build-storybook:gatsby` to build storybook with conditional font path
`yarn build:gatsby && yarn serve:gatsby` to build doc site in production mode and serve it

Notice that before this change, you can see console errors about missing font files from storybook
<img width="1787" alt="Screen Shot 2022-05-13 at 11 53 02 AM" src="https://user-images.githubusercontent.com/33579665/168340487-66b87330-7dc0-42de-aa6f-0ad23767f9bf.png">

After change, no console errors and font files can be seen in network tab
<img width="1787" alt="Screen Shot 2022-05-13 at 11 47 34 AM" src="https://user-images.githubusercontent.com/33579665/168340544-ad2f2efe-b68a-47db-87ad-c49093cde1d7.png">
